CHANGELOG.md

```diff
@@ -7,10 +7,12 @@
 * Removed deprecated save() method from RuntimeConfig class
 * Removed is_saved() method and "name" setter from RuntimeConfig class
 * WorkflowRun.get_results() returns values consistent with vanilla python - single results are returned as-as, multiple results are returned as a tuple.
+* If you set a custom task image (`@sdk.task(custom_image=...)`), the local Ray runtime won't execute the task unless the local Ray cluster was started with custom resources.
```

Comment:
Suggestion: Please do add a jira ticket to do something with that, either warn a user or change the behaviour to ignore custom image locally entirely.

Reply:
The follow up ticket was mentioned in the PR description: https://zapatacomputing.atlassian.net/browse/ORQSDK-838

I didn't add it to the changelog because I don't think it fits here.